### DISCUSSION TITLE
Future core competency of clinical statistical programmer with AI/LLM

### DISCUSSION THREAD

**Discussion by lengning** [rinpharma/rinpharma-summit-2024#4] [roundtable]
### Proposal

A pressing topic I think we should discuss is......
...

### Expected impact

I believe this topic would be a benefit because......
...

### Prior discussions/work

Some prior work includes......
...

### Would you be willing to potentially facilitate this discussion?

None

**Comment by harveyl888** [comment]
I was thinking of the same topic - it leads to the question of how much of the statistical programmer's work will be replaced by AI?

**Comment by telepath37** [comment]
It may be worth to discuss some prototypes we pilot at Roche (i.e. NEST chatbot) and how we envison the evolution of 'programmer' roles in trial analysis space

**Comment by epijim** [comment]
Good discussion at the advisory board meeting (29th May), saying can there be a proposal about what solutions could be? Can we reword titles like this to be questions?

**Comment by katieigartua** [comment]
One discussion point we've had was creating a cheatsheet of R tools and best practices that could be shared to the community as a delivery.  

Feel free to share your experience with LLMs in R here as discovery!

**Comment by epijim** [comment]
This has been added to the agenda for the summit
